doc/cygnus-ngsi/flume_extensions_catalogue/orion_ckan_sink.md

```diff
@@ -354,3 +358,45 @@ In detail, this is the workflow when `NGSICKANSink` is combined with `CKANCache`
 
 [Top](#top)
 
+##<a name="section4"></a>Annexes
+
+###<a name="section1"></a>Provisioning a CKAN resource for the column mode
+First of all, you'll need a CKAN organization and package/dataset before creating a resource and an associated datastore in order to persist the data.
+
+Creating an organization, let's say in [`demo.ckan.org`](http://demo.ckan.org/); the organization name is `service`, because our entity will be in that FIWARE service:
```

Comment:
I would add: "...let's say in demo.ckan.org for the sake of demonstration, but should be a CKAN deployment of yours;..."